buddyboss/buddyboss-platform
# Wordpress Search function not working

Added code to support global search for page and other custom post types

### All Submissions:




### Changes proposed in this Pull Request:

Added search support for page and other custom post types. 

### How to test the changes in this Pull Request:

1. Click on 'search' icon in top navigation menu bar
2. Type in search term with page or Custom CPT like course, lesson etc.
3. See error - no search results returned.

### Proof Screenshots or Video

<img width="1351" alt="Screenshot 2020-09-07 at 8 30 38 PM" src="https://user-images.githubusercontent.com/2979084/92400121-0068db80-f149-11ea-9951-17722c4640ae.png">

<img width="1177" alt="Screenshot 2020-09-07 at 8 28 16 PM" src="https://user-images.githubusercontent.com/2979084/92400135-078fe980-f149-11ea-96d6-8061ec0853fc.png">

### Other information:




### Changelog entry

> Added search support for page and other custom post types

.
